.github/workflows/tests.yml

```diff
@@ -90,6 +87,7 @@ jobs:
         if: ${{ matrix.impl == 'c' }}
         run: |
           echo "DEPS=$DEPS ./psycopg_c" >> $GITHUB_ENV
+          sudo apt-get install -y libgeos-dev
```

Comment:
Worked (but maybe it is a wrong place to have it here), but another error occurred: ModuleNotFoundError: No module named 'distutils.msvccompiler' ([link](https://github.com/psycopg/psycopg/actions/runs/12471615165/job/34808906125?pr=977))

Reply:
@dvarrazzo , so, is it OK to leave here or there is a better place for it?

Reply:
No, we don't need it. The error only appeared because the wheel package for shapely 1.7.0 is not available for Python 3.9, therefore pip tried to install the requested version from source, and failed on the missing library.

Now that we are using a minimum version for which there is a wheel package for Python 3.9 we don't need this library.

Reply:
Gotcha, understood, removed. Thank you for your patience and explanations.